app/assets/javascripts/browsered/web-payments/google-pay.js

```diff
@@ -39,6 +44,95 @@ const processPayment = paymentData => {
     })
 }
 
+const checkOriginHostName = origin => {
+  return Charge.worldpay_3ds_flex_ddc_url.lastIndexOf(origin, 0) === 0
+}
+
+const performDeviceDataCollectionForGooglePay = (paymentData) => {
+  const deviceDataCollectionFinishedEventListener = event => {
+    if (checkOriginHostName(event.origin) && event.source === iframeWindow) {
+      const { MessageType, SessionId, Status } = JSON.parse(event.data)
+      if (MessageType === 'profile.completed') {
+        var ddcStatus = null
```

Comment:
as this is always set, no need to initalise 

```suggestion
        var ddcStatus
```

Reply:
👍